### PR TITLE
Drn 106 change links

### PR DESCRIPTION
**Pull request by mattrobertsky**

### CODE REVIEW

File: app/uk/gov/hmrc/disguisedremunerationfrontend/views/answer_table.scala.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                @for(k <- headingKeys) {
+                    <th class="govuk-table__header" scope="col">@messages(k)</th>
+                }
+            </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+        @for((_,row) <- rows) {
+            <tr>
+                @for(cell <- row) {
+                    @*TODO remove BR once UX have checked design*@
```
Comment (adamdye-hmrc) [inline app/uk/gov/hmrc/disguisedremunerationfrontend/views/answer_table.scala.html]: Remove this TODO?

File: app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala [hunk 2]
```diff
@@ -380,66 +380,115 @@ class JourneyController @Inject()(
...
-              contactDetails.address.lines.
-                map(escape).
-                intercalate(Html("<br />")),
-            msg("contact-details") -> {
+      case JourneyState(Some(aboutYou), _, Some(contactDetails)) =>
+        views.html.answer_list(
+          msg("personal-details"),
+          List(
+            (msg("name"), escape(username), None),
+            (msg("filling-in-form-for-self"), msg(if(aboutYou.completedBySelf) "TRUE" else "FALSE"), "about-you/about-you".some),
```
Comment (adamdye-hmrc) [inline app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala]: We should have more details from the aboutYou/Self journeys here.

Comment (mattrobertsky) [reply app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala]: That should be a new ticket, I've just added the change link to the existing details.

File: app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala [hunk 3]
```diff
@@ -380,66 +380,115 @@ class JourneyController @Inject()(
...
+              (msg("tax-or-national-insurance-paid-or-agreed-to-pay"),
+                settlement.fold(msg("none")){x => Html(s"£${x.amount}")},
+                settlement.fold(s"scheme/$index/tax-settled")(_ => s"scheme/$index/add-settlement").some )
             )
+          ) |+|
+          views.html.answer_table(Html(s"$name"), loanDetailsTableHeadings, scheme.loanDetails.flatMap(
+            {
+              case(k,v) => {
+                Map(k.toString -> v.fold(List.empty[Html]){ ld =>
+                  ld.rowValues(index, k.toString):+ Html(s"""<a href="scheme/$index/details/${k.toString}/loan-amount">${msg("tasklist.edit")}</a>""")
```
[the remark this answers is not in the document]
Comment (mattrobertsky) [reply app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala]: Ah, is that the first page in the journey?

File: app/uk/gov/hmrc/disguisedremunerationfrontend/views/answer_table.scala.html (hunk 1 above)
Comment (mattrobertsky) [reply app/uk/gov/hmrc/disguisedremunerationfrontend/views/answer_table.scala.html]: Good point.

File: app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala (hunk 3 above)
[the remark this answers is not in the document]
Comment (adamdye-hmrc) [reply app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala]: Yes it is

File: app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala (hunk 2 above)
Comment (adamdye-hmrc) [reply app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala]: Fair enough, Yordan is going to make a new ticket.

**Pull request merged**